backend/services/transactions/src/routes.py

```diff
@@ -1,5 +1,127 @@
 from src import app
+from flask import request, jsonify
+from ecdsa import SigningKey, VerifyingKey
+import base64
+
+import sys, os
+sys.path.append(os.path.abspath(os.path.join('..', '')))
+
+from shared import HttpCode
+## Global
+COINBASE_REWARD = 10
+INCORRECT_PAYLOAD_MSG = "Please send correct json payload"
+VERFICATION_FAILURE= "Unable to Verify the Transaction"
+
+## Temporary
+TransactionPool = []
+private_keys = {}
+public_keys = {}
+
+
+def validateSignature(id, signature, address):
```

Comment:
Keep this and createTransaction() minus its key-related stuff and take out all other endpoints